### DISCUSSION TITLE
in game alternative song chooser

### DISCUSSION THREAD

**Discussion by Konpaku-LVX** [ppy/osu#13411] [Ideas] [answered]
this is just a wild idea i had randomly but i think its really cool if its somehow possible to pull off,
i was thinking maybe it was possible to have a feature to search up music off of some service like funkwhale or youtube or deezer inside the client (having osu!lazer ui for the search maybe something looking like the beatmaps page) to temporarily use a different song for that map, lets say youre playing a map and its a cover and you wanna play the original song, or if youre playing an old map that doesn't even have audio anymore, without something like this you would have to manually find the mp3 and replace it in your game, then make a local offset so it works, which is pretty hard or just annoying sometimes to do, and hard to do it right if you dont have a perfect sense of setting your offset. and also of course this is just an idea that came to mind aswell

**Comment by peppy** [comment, marked as answer]
Never going to happen sorry. It's out of scope of this project.

**Reply by Konpaku-LVX** [reply]
yeah i assumed so -w- it was a pretty ambitious idea. was still a pretty cool idea though right? 😋
